Services/SelectedStopPointManager.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace CanalTP\MttBundle\Services;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Symfony\Component\Serializer\Serializer;
+
+use CanalTP\MttBundle\Entity\LineTimetable;
+use CanalTP\MttBundle\Entity\SelectedStopPoint;
+
+/**
+ * Class SelectedStopPointManager
+ * @package CanalTP\MttBundle\Services
+ */
+class SelectedStopPointManager
+{
+    private $om = null;
+    private $repository = null;
+    private $serializer = null;
+    private $navitia = null;
+
+    /**
+     * Constructor
+     *
+     * @param ObjectManager $om
+     */
+    public function __construct(ObjectManager $om, Navitia $navitia, Serializer $serializer)
```

Comment:
I think I will use snake case in json keys and we will be able to call it in other services if needed. For the moment its use may be overkill but it can be good to have it available later.